javascript-modal/styles.css

```diff
@@ -0,0 +1,80 @@
+html {
+  margin: 0;
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  min-height: 100%;
+}
+
+/* doesnt change */
+.open-btn {
+  background-color: blue;
+  color: white;
+  border: 0;
+  font-size: 1.5rem;
+  padding: 0.8rem 1.5rem;
+  margin: 0.5rem;
+  border-radius: 1rem;
+  position: absolute;
```

Comment:
Why are you giving the button `position: absolute`? It doesn't need to be absolutely positioned, it can just sit in the normal document flow.

Reply:
Styled the background to the .column class instead of container, button isn't getting covered by the overlay. Noted for next time to throw the overlay in the parent container.